src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,73 @@
+package duke;
+
+import duke.command.Command;
+import duke.task.TaskList;
+import javafx.application.Platform;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+/**
+ * Main class to run Duke program.
+ * Creates and stores your task list.
+ */
+public class Duke {
+
+    private Storage saveData;
```

Comment:
hmm, i feel its confusing to use verb as a start of variable name, as normally we use verb to start a method name.